### PR TITLE
Bump Version, New Blocks , Items 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by buchwasa** [comment]
Because changing the server version automatically updates everything

**Comment by KurojakiKaoru** [comment]
Lol, this should be closed.

File: src/main/java/cn/nukkit/item/Item.java [hunk 1]
```diff
@@ -995,10 +998,22 @@ private static void initCreativeItems() {
         addCreativeItem(Item.get(Item.FENCE_GATE_ACACIA, 0));
         addCreativeItem(Item.get(Item.FENCE_GATE_DARK_OAK, 0));
         addCreativeItem(Item.get(Item.IRON_BARS, 0));
-        int[] dyeColors = {0, 8, 7, 15, 12, 14, 1, 4, 5, 13, 9, 3, 11, 10, 2, 6};
-        for (int color : dyeColors) {
-            addCreativeItem(Item.get(Item.BED, color));
-        }
+        addCreativeItem(Item.get(Item.BED, DyeColor.WHITE.getWoolData()));
```
Comment (ghost) [inline src/main/java/cn/nukkit/item/Item.java:1001]: @PikyCZ This method it's depredicated :(

Comment (ghost) [reply src/main/java/cn/nukkit/item/Item.java:1001]: but its not working .. ::

Comment (ghost) [reply src/main/java/cn/nukkit/item/Item.java:1001]: Wait.

File: src/main/java/cn/nukkit/block/Block.java [hunk 2]
```diff
@@ -802,6 +805,10 @@ public final int getDamage() {
     public final void setDamage(Integer meta) {
         this.meta = (meta == null ? 0 : meta & 0x0f);
     }
+    
+    public int getMaxStackSize() {
```
Comment (Creeperface01) [inline src/main/java/cn/nukkit/block/Block.java:809]: Wtf?

Comment (ghost) [reply src/main/java/cn/nukkit/block/Block.java:809]: Delete

File: src/main/java/cn/nukkit/item/Item.java (hunk 1 above)
Comment (ghost) [reply src/main/java/cn/nukkit/item/Item.java:1001]: @PikyCZ Use one arraylist for metadata and simple add "addCreativeItem(Item.get(Item.BED, metadata);

**Pull request closed without merging**